### DISCUSSION TITLE
Is it really not possible to upload folders to storage via studio?

### DISCUSSION THREAD

**Discussion by kassim3** [supabase/supabase#14187] [Questions] [answered]
Is it possible to upload folders instead of manually uploading one file at a time. I'm currently moving a project to supabase and i want to move around 1000 images. I really don't want to upload it one file at a time. Am i being stupid?

**Comment by GaryAustin1** [comment]
In windows you can drag a picture folder and drop it here:
![image](https://user-images.githubusercontent.com/54564956/236706090-274dcf6a-62e7-4a8d-ab52-f2bc61d0fb37.png)

You can also click upload and select many files and they will be uploaded.

**Reply by kassim3** [reply]
Ok that might be the issue. I'm using fedora with gnome and using chrome. I tried selecting multiple folders and it doesn't do anything. Clicking on a single folder and selecting open just opens the folder. I'll give the drag and drop a try hopefully that will work 🤞

**Reply by GaryAustin1** [reply]
Using upload and clicking a folder does just open it, but then you can shift-select as many files as you want, at least on windows.
Dragging a folder from file explorer into the red area, raises an input box for dropping the file or folder.

**Reply by kassim3** [reply]
> Using upload and clicking a folder does just open it, but then you can shift-select as many files as you want, at least on windows. Dragging a folder from file explorer into the red area, raises an input box for dropping the file or folder.

so selecting multiple folders doesn't work as well as drag and dropping multiple folders. Think this might be an OS issue

**Reply by kassim3** [reply, marked as answer]
Firefox worked!!!!
